katdal/chunkstore_s3.py

```diff
@@ -338,16 +359,35 @@ def create_array(self, array_name):
             with self._request(None, 'PUT', policy_url, data=json.dumps(policy)):
                 pass
 
+    @classmethod
+    def _numpy_header(cls, chunk):
+        fp = io.BytesIO()
+        header_fields = np.lib.format.header_data_from_array_1_0(chunk)
+        np.lib.format.write_array_header_1_0(fp, header_fields)
```

Comment:
I used to wonder about forcing the headers to v1 in my previous attempts at this... I guess we would not have overly elaborate dtypes in our chunk store, so v2 headers would be overkill (and less compatible).

Reply:
Yes I seem to remember us having discussions about this. I can't remember what we settled on the first time around. I figure it's a problem we can solve if we ever actually run into it.